src/darsia/utils/segmentation.py

```diff
@@ -33,9 +34,12 @@ def segment(
         edges_method (str): "gradient_based" or "scharr", deciding which algorithm
             is used for determining edges; the former uses gradient filtering while
             the latter uses the Scharr algorithm.
+        mask (np.ndarray, optional): binary array, only where true, segmentation is
+            performed
         verbosity (bool): flag controlling whether relevant quantities are plotted
             which is useful in the tuning of the parameters; the default is False.
         keyword arguments (optional): tuning parameters for the watershed algorithm
+            "method" (str): 'median' or 'tvd'
```

Comment:
I am not sure whether it matters, but you could consider adding that tvd is anisotropic.